sw/airborne/modules/sonar/sonar_bebop.c

```diff
@@ -206,20 +225,21 @@ void *sonar_bebop_read(void *data __attribute__((unused)))
 #else // SITL
       sonar_bebop.distance = stateGetPositionEnu_f()->z;
       Bound(sonar_bebop.distance, 0.1f, 7.0f);
-      sonar_bebop.meas = sonar_bebop.distance / SONAR_BEBOP_INX_DIFF_TO_DIST;
+      sonar_bebop.meas = (uint16_t)(sonar_bebop.distance / SONAR_BEBOP_INX_DIFF_TO_DIST);
 #endif // SITL
 
       // Send ABI message
       uint32_t now_ts = get_sys_time_usec();
       AbiSendMsgAGL(AGL_SONAR_ADC_ID, now_ts, sonar_bebop.distance);
-#ifdef SENSOR_SYNC_SEND_SONAR
-      // Send Telemetry report
-      DOWNLINK_SEND_SONAR(DefaultChannel, DefaultDevice, &sonar_bebop.meas, &sonar_bebop.distance);
-#endif
+
 #ifndef SITL
+    #ifdef SENSOR_SYNC_SEND_SONAR
```

Comment:
why not making this message available even in SITL ?

Reply:
Yes, good idea

Reply:
....Will then also enable SONAR_COMPENSATE_ROTATION and SONAR_BEBOP_FILTER_NARROW_OBSTACLES to be used in SITL